Jsonld files are not registered (Windows only)

# Bug Report

## Describe the Bug

When starting EDC with Windows, lson ld files are not registered. Log files read:

```
INFO 2024-05-28T12:02:01.6806667 Initialized JSON-LD Extension
WARNING 2024-05-28T12:39:12.9628396 Failed to register cached json-ld document: Cannot find resource document\odrl.jsonld
WARNING 2024-05-28T12:39:12.966752 Failed to register cached json-ld document: Cannot find resource document\dspace.jsonld
```

Later when trying to create a policy, an error message is shown (return code 400):

`Caused by: JsonLdError[code=The document could not be loaded or parsed [code=LOADING_DOCUMENT_FAILED]., message=URL scheme [http] is not supported.]
`
### Expected Behavior

Warning does not appear. Creating policy is possible.

### Observed Behavior

See description

## Steps to Reproduce

Steps to reproduce the behavior:

Just start a EDC connector (e.g. Sample project for simple transfer) with Windows.

## Context Information

EDC 0.7.0, Windows 10

## Possible Implementation

I found the root cause already: 

In  ([org.eclipse.edc.jsonld.JsonLdExtension:96](https://github.com/eclipse-edc/Connector/blob/7b022912d45a8537c2d39cf07de0c8464d945274/extensions/common/json-ld/src/main/java/org/eclipse/edc/jsonld/JsonLdExtension.java#L96))

`).forEach(jsonLdContext -> getResourceUri("document" + **File.separator** + jsonLdContext.fileName())
`

"File.separator" must not be used. Its backslash for Windows. Uri need to have "/" only.


Furthermore there is a syntax error in [dspace.jsonld](https://github.com/eclipse-edc/Connector/blob/7b022912d45a8537c2d39cf07de0c8464d945274/extensions/common/json-ld/src/main/resources/document/dspace.jsonld#L58):

`Invalid token=STRING at (line no=59, column no=19, offset=2480). Expected tokens are: [COMMA]
`

--- a/extensions/common/json-ld/src/main/java/org/eclipse/edc/jsonld/JsonLdExtension.java
+++ b/extensions/common/json-ld/src/main/java/org/eclipse/edc/jsonld/JsonLdExtension.java
@@ -93,7 +93,7 @@ public class JsonLdExtension implements ServiceExtension {
         Stream.of(
                 new JsonLdContext("odrl.jsonld", "http://www.w3.org/ns/odrl.jsonld"),
                 new JsonLdContext("dspace.jsonld", "https://w3id.org/dspace/2024/1/context.json")
-        ).forEach(jsonLdContext -> getResourceUri("document" + File.separator + jsonLdContext.fileName())
+        ).forEach(jsonLdContext -> getResourceUri("document/" + jsonLdContext.fileName())
                 .onSuccess(uri -> service.registerCachedDocument(jsonLdContext.url(), uri))
                 .onFailure(failure -> monitor.warning("Failed to register cached json-ld document: " + failure.getFailureDetail()))
         );
@@ -131,6 +131,7 @@ public class JsonLdExtension implements ServiceExtension {
         }
     }
 
-    record JsonLdContext(String fileName, String url) { }
+    record JsonLdContext(String fileName, String url) {
+    }
 
 }

